dimagi/pages/templates/svg/pillar/dc/chapter3/1.html

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="68px" height="68px" viewBox="0 0 68 68" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linejoin="round">
+        <g id="Group-3" transform="translate(10.000000, 3.000000)" stroke="#004EBC" stroke-width="2">
+            <path d="M24.3692308,52.1605113 L42.0923077,52.1605113 L24.3692308,52.1605113 Z M5.90769231,55.8663937 L17.7230769,55.8663937 L17.7230769,46.972276 L5.90769231,46.972276 L5.90769231,55.8663937 Z M24.3692308,32.8899231 L42.0923077,32.8899231 L24.3692308,32.8899231 Z M24.3692308,13.6193349 L42.0923077,13.6193349 L24.3692308,13.6193349 Z M5.90769231,17.1036055 L17.7230769,17.1036055 L17.7230769,8.2094878 L5.90769231,8.2094878 L5.90769231,17.1036055 Z M37.1121231,0.056546628 L47.4624,10.4359819 L37.1121231,0.056546628 Z M0,2.13162821e-14 L36.9230769,2.13162821e-14 L36.9230769,9.91345251 C36.9304615,10.2692172 37.1874462,10.6472172 37.6615385,10.654629 L47.4701406,10.654629 L47.4701406,62.5369819 L0,62.5369819 L0,2.13162821e-14 Z" id="Stroke-1"></path>
+            <polyline id="Path-10" stroke-linecap="round" points="13.8179477 26.7388996 5.90769231 26.7388996 5.90769231 38.4428592 17.6568204 38.4428592 17.7230769 36.6178654"></polyline>
+            <polyline id="Path-9" stroke-linecap="round" points="11.0769231 31.6074523 14.7729649 34.0818709 20.7912964 25.2565466"></polyline>
+        </g>
+    </g>
+</svg>
```

Comment:
make sure there are new lines at the end of these svg files. also, please make sure the indentation is 2 spaces, not 4